packages/headless/src/api/service/insight/insight-api-client.ts

```diff
@@ -112,4 +114,8 @@ export class InsightAPIClient {
       ? {success: body as InsightUserActionsResponse}
       : {error: body as InsightAPIErrorStatusResponse};
   }
+
+  async html(req: HtmlRequest) {
```

Comment:
The insight-api-client is meant to interact with the insight api. So I don't think we should be putting this in here. I would be more keen on making just removing quickview capability until the proxy route exists.

Reply:
You are right! I'll close this PR then. Let's add the html method in the right place when the routes for the Quick view are ready.